### PR TITLE
Improve check_opengraph()

### PR DESCRIPTION
**Pull request by hadley**
* Show path to make it easier to see where the problem is coming from
* Use our standard display of yaml variables
* Other various minor tidyups

### CODE REVIEW

**Comment by jayhesselberth** [comment]
Looks good to me. @gadenbuie originally added open graph support and may want to weigh in.

**Review by jayhesselberth — APPROVED** [review]
Looks good to me. @gadenbuie originally added open graph support and may want to weigh in.

File: R/render.R [hunk 1]
```diff
@@ -153,54 +153,35 @@ data_open_graph <- function(pkg = ".") {
...
-        "The {.var opengraph: twitter} option must be a list.",
-        call = caller_env()
-      )
-    }
+    check_open_graph_list(og$twitter, path, paste0(base_path, ".twitter"), call)
     if (is.null(og$twitter$creator) && is.null(og$twitter$site)) {
       cli::cli_abort(
-        "{.var opengraph: twitter} must include either {.val creator} or {.val site}.",
-        call = caller_env()
+        "{.var opengraph.twitter} must include either {.var creator} or {.var site}.",
```
Comment (hadley) [inline R/render.R:178]: Need to use `.field`

**Comment by gadenbuie** [comment]
Thanks for tagging me @jayhesselberth, I took a quick look and nothing jumped out at me. Looks great! Showing the path where the error happens will be very helpful.

**Pull request merged**